dipy/denoise/localpca.py

```diff
@@ -133,4 +152,5 @@ def localpca(arr, sigma, patch_radius=2, tau_factor=2.3, out_dtype=None):
 
     denoised_arr = thetax / theta
     denoised_arr.clip(min=0, out=denoised_arr)
+    denoised_arr[~mask] = 0
```

Comment:
Nope, this is an application of the mask. If we put back the original values we will have noise in the background which is undesired.